instances/technique/bootstrapAggregating.jsonld

```diff
@@ -0,0 +1,18 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/bootstrapAggregating",
```

Comment:
@skoehnen I still have an issue with this term... It sounds incomplete (although I know that this term is used that way). Would it be incorrect to call it "bootstrap aggregation" or "bootstrap aggregating analysis" or just "bagging" ?